src/index.html

```diff
@@ -5,10 +5,70 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
+
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="moyo-logo-link">
+        <img class="moyo-logo"
+        src="/images/logo.png"
+        alt="moyo logotype"
+        >
+      </a>
+
+      <nav class="header-nav">
```

Comment:
such a menu can also be used in the footer, so in this case it is better not to bind the class name to the place where this element is used
`<nav class="nav">` will be better